bldr/data/hooks/pre-install-deps

```diff
@@ -1 +1,27 @@
 #!/bin/bash
+
+perform_unminimization() {
+    unminimize_cmd=$(command -v unminimize)
```

Comment:
local

Reply:
Part of why I recommended dropping `unminimize_present` altogether. But you can make the assignment in a separate command (right after `local`) to work around that.